docs/api-docs/provider-apis/data-solutions/data-solutions-overview.md

```diff
@@ -0,0 +1,539 @@
+
+```JSON title="Sample Request" lineNumbers
+PUT https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/settings/analytics/{id}
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+  "id": 6,
+  "channel_id": 0,
+  "name": "Site Verification Tags",
+  "enabled": false,
+  "verification_tag": "<meta name=\"google-site-verification\" content=\"twLOPnFqwX5JK2-RksTZp_QEkfUPPgfGJ0_UHrEKLcY\" />"
+}
+
+```
+
+```JSON title="Sample Response" lineNumbers
+{
+    "data": {
+        "id": 6,
+        "channel_id": 0,
+        "name": "Site Verification Tags",
+        "enabled": false,
+        "verification_tag": "<meta name=\"google-site-verification\" content=\"twLOPnFqwX5JK2-RksTZp_QEkfUPPgfGJ0_UHrEKLcY\" />"
+    },
+    "meta": {}
+}
+```
+
+&nbsp;
+
+## Affiliate Conversion Tracking
+
+| Field | Type | Description |
+| ------------- | -------- |  -------- |
+| id | integer | Id of the Web Analytic. `1` for Affiliate Conversion Tracking |
+| channel_id | integer | Id of the storefront channel. Default is `0`. |
+| name | string | Name of Web Analytic `Affiliate Conversion Tracking` |
+| enabled | boolean | Indicates whether merchant has [enabled Affiliate Conversion Tracking](https://support.bigcommerce.com/s/article/Data-Solutions?language=en_US#web-analytics) in their store |
+| connection | string | Affiliate Conversion Tracking Code |
+
+### Get the Affiliate Conversion Tracking Analytic
+
+If a merchant has not entered an Affiliate Conversion Tracking Code, `connection` will return as an empty string.
+
+```JSON title="Sample Request" lineNumbers
+GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/settings/analytics/{id}
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+```
+
+```JSON title="Sample Response" lineNumbers
+{
+  "id": 7,
+  "channel_id": 0,
+  "name": "Affiliate Conversion Tracking",
+  "enabled": false,
+  "connection": "<script>js code here...</script>"
+}
+```
+
+### Update the Affiliate Conversion Tracking Analytic
+
+```JSON title="Sample Request" lineNumbers
+PUT https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/settings/analytics/{id}
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+    "data": {
+        "id": 7,
+        "channel_id": 0,
+        "name": "Affiliate Conversion Tracking",
+        "enabled": false,
+        "connection": "<script>js code here...</script>"
+    },
+    "meta": {}
+}
+
+```
+
+```JSON title="Sample Response" lineNumbers
+{
+    "data": {
+        "id": 7,
+        "channel_id": 0,
+        "name": "Affiliate Conversion Tracking",
+        "enabled": false,
+        "connection": "<script>js code here...</script>"
+    },
+    "meta": {}
+}
+```
+
+&nbsp;
+
+## Get All Web Analytics
+
+All six web analytics will be returned. As shown, fields for codes (e.g. `verification_tag`) in which a merchant has not entered a value will return as an empty string.     
```

Comment:
Recommended to use "such as" or "for example" instead of "e.g." https://developers.google.com/style/word-list#eg